Dockerfile.dapper

```diff
@@ -1,51 +1,61 @@
 FROM ubuntu:16.04
-# FROM arm=armhf/ubuntu:16.04
 
 ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPER_ENV=TAG \
-    DAPPER_SOURCE=/go/src/github.com/submariner-io/submariner DAPPER_DOCKER_SOCKET=true \
-    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} DAPPER_OUTPUT=output \
-    LINT_VERSION=v1.16.0 \
-    HELM_VERSION=v2.14.1 \
-    KIND_VERSION=v0.3.0 \
-    KUBEFED_VERSION=0.1.0-rc2 \
-    GO_VERSION=1.12.6
+    DAPPER_SOURCE=/go/src/github.com/submariner-io/lighthouse DAPPER_DOCKER_SOCKET=true \
+    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} DAPPER_OUTPUT=output
 
 RUN rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go GO111MODULE=on PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash GOFLAGS=-mod=vendor
 
 # Requirements:
-# Component      | Usage
+# Component       | Usage
 # -----------------------------------------------------------
-# gcc            | ginkgo
-# git            | find the workspace root
-# curl           | download other tools
-# docker.io      | Dapper
-# golang         | build
-# kubectl        | e2e tests
-# golangci-lint  | code linting
-# helm           | e2e tests
-# kubefedctl     | e2e tests
-# kind           | e2e tests
-# ginkgo         | tests
-# goimports      | code formatting
+# gcc             | ginkgo
+# git             | find the workspace root
+# curl            | download other tools
+# docker.io       | Dapper
+# build-essential | coredns make
```

Comment:
Do we need `build-essential` here, or only `make`? `build-essential` pulls in `g++` and `dpkg-dev` along with `make`, which adds quite a lot to the container.

Reply:
Good point.

Reply:
Added just make and tested.